driver/reducecalculation_driver.hpp

```diff
@@ -2,7 +2,7 @@
  *
  * MIT License
  *
- * Copyright (c) 2023 Advanced Micro Devices, Inc.
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
```

Comment:
should not change this if you just modify this file

Reply:
I'm not sure about this. I changed it because I thought we should change it to the lastest time of file modification. But I remove the change as you suggested